### DISCUSSION TITLE
Is it safe to terminate these processes to save CPU?

### DISCUSSION THREAD

**Discussion by renatopinto89** [AzuraCast/AzuraCast#5583] [Q&A]
I've been experiencing some random CPU spikes on my AzuraCast installations, that appear to trigger after a station restart (Restart Broadcasting).
They seem to be log related so I'm not sure if they are completely necessary, so my question is if it is safe to terminate though htop without completely breaking the container.
0.17.2 Stable Channel - Docker

Thank you

![IMG_20220709_150721](https://user-images.githubusercontent.com/83221826/178109511-b47cf44d-8659-4709-9926-75dfaf3936ae.jpg)

**Comment by Mozitux** [comment]
Good evening, I already had some problems when I abruptly stopped the Azuracast process. I'm not saying it will necessarily happen to you, but I recommend that you avoid killing the latter.

If your CPU usage is too high, you can consider going to your station profile, AUTODJ tab and setting the "Liquidsoap performance optimization" option to "Use less CPU (uses more memory)".

**Comment by Sakretsos** [comment]
@renatopinto89 Hello, try to use Rolling Release channel, since the issue have been already fixed there.
